src/jvm/test/org/jetbrains/kotlinx/lincheck_test/AbstractLincheckTest.kt

```diff
@@ -67,4 +67,4 @@ abstract class AbstractLincheckTest(
     }
 }
 
-private const val TIMEOUT = 20 * 60_000L // 10 min
\ No newline at end of file
+private const val TIMEOUT = 20 * 60_000L // 20 min
```

Comment:
Why this is necessary?

Reply:
Because 1 minute is `60_000L` milliseconds, and `20 * 60_000L` is 20 minutes, not ten.